modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc

```diff
@@ -0,0 +1,130 @@
+// CNF-950 4.7 Provisioning and deploying a Distributed Unit (DU) manually
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-provisioning-deploying-a-distributed-unit-manually_{context}"]
+= Provisioning and deploying a Distributed Unit (DU) manually
+
+Radio access network (RAN) is composed of Centralized Units (CU), Distributed Units (DU) and Radio Units (RU).
```

Comment:
(DU), and 

(Oxford comma)

Also, I wonder if each of these units can be in lowercase.

Reply:
Approved.